frontend/src/components/Logs/RawLogView/index.tsx

```diff
@@ -37,12 +37,19 @@ const convert = new Convert();
 interface RawLogViewProps {
 	isActiveLog?: boolean;
 	isReadOnly?: boolean;
+	isTextOverflowEllipsisDisabled?: boolean;
 	data: ILog;
 	linesPerRow: number;
 }
 
 function RawLogView(props: RawLogViewProps): JSX.Element {
-	const { isActiveLog = false, isReadOnly = false, data, linesPerRow } = props;
+	const {
```

Comment:
As we are refactoring the code, can you please destructure the props at the parameter level? 
```suggestion
	function RawLogView({
		isActiveLog = false,
		isReadOnly = false,
		data,
		linesPerRow,
		isTextOverflowEllipsisDisabled = false
	}: RawLogViewProps): JSX.Element {
```